src/pages/workspaces/List.js

```diff
@@ -258,7 +258,7 @@ export const WorkspaceList = () => {
                   closeOnClick: true,
                   content: h(WorkspaceMenuContent, { namespace, name, onShare, onClone, onDelete })
                 }, [
-                  h(Link, { 'aria-label': 'Workspace menu', disabled: !canView }, [icon('cardMenuIcon', { size: 20 })])
+                  h(Link, { 'aria-label': 'Workspace menu', disabled: !canView, 'aria-hidden': false }, [icon('cardMenuIcon', { size: 20 })])
```

Comment:
TOL: It's very interesting to see these 2 changes in the same PR. Having to explicitly say `'aria-hidden': false`, especially at the point of use (rather than baked into a reusable component) feels less than ideal. I'm not sure what the right solution is, though.

Maybe `PopupTrigger` should have `role: 'button'` like `Interactive` (conditionally) does. Maybe `Link` (or `Clickable` or `Interactive`) should automatically add `'aria-hidden': false` when an `aria-label` is present.

Do others on the team have thoughts?

Reply:
This piece I am not 100% sure on. In the video and in the acceptance criteria in the ticket these 3-dot menus are called out as lacking sufficient screen-reader transparency. However I was not finding the 3-dot menus to be problematic so I figured it might be a browser configuration thing and that setting aria-hidden to false rather than depending on it implicitly setting to false could alleviate that discrepancy.

Reply:
There's also the question of whether or not the aria-label needs to be more specific, i.e. naming the workspace it applies to. The ticket references the old design in which it's clear that there is no other context, such as a table column header, to help disambiguate. In the new design, that is less clear (though from what I remember of the implementation, I think these aren't technically table columns).